### ISSUE TITLE
[SUGGESTION] installation as a Homebrew package?

### ISSUE THREAD

**Issue by vincentneo** [sketch-hq/fig2sketch#61]
Thanks for making this tool! Great work!

Though I think it would be easier if this is also available to download via homebrew as well, easier to handle updates, etc too for the end-user.

What do you guys think?

**Comment by javitonino** [comment]
Hi! and thanks for your kind words :heart: 

Better packaging is definitely in our plans for the next few months. We are still not sure if homebrew would be the best option for this, given it's a Python package we may use Python packages instead, which might fit better and has a lot of the same advantages.

**Comment by vincentneo** [comment]
> given it's a Python package we may use Python packages instead

Right! that does make sense too.

**Comment by javitonino** [comment]
Hey, forgot to mention last week before I went for a short holiday, but we have already released on `pip` and updated the README with instructions. 

For now, we'll keep it like this. We may consider releasing on brew later, but the next step on packaging should probably be to get our mac binaries signed/notarized. For now, I think pip is enough to keep updated without too much of a hassle :crossed_fingers:
